src/applications/mhv-landing-page/tests/e2e/alerts/unverified.cypress.spec.js

```diff
@@ -39,14 +39,13 @@ describe(appName, () => {
           cy.findByRole('heading', { name }).should('not.exist');
         });
 
-        // Test the hubs are visible
-        pageLinks.hubs.forEach(hub => {
-          LandingPage.validateLinkGroup(hub.title, hub.links.length);
+        // Test the cards are not visible
```

Comment:
Note: This E2E test is for an unverified (LOA1) user, but the landing page was showing all the cards the way the previous code was setup. This was due to a check for just the registered flag which in production cannot be true if the user is unregistered. This PR now makes sure that an LOA1 user is not a registered user. Note this only affected the test and not production.